preconditions/iotics_test.sh

```diff
@@ -1,21 +1,20 @@
 #!/bin/bash
 
 CURRENT_DIR=$(pwd)
-SPACE_NAME=${1:-ganymede}
-HTTPS_PREFIX="https://"
-IOTICSPACE_DOMAIN="$HTTPS_PREFIX$SPACE_NAME.iotics.space"
+SPACE_NAME=${1:-"demo"}
```

Comment:
on second thought why do you have the host_Id?
if you want to check for the resolver why isn't this enough

`curl --head https://did.prd.iotics.com/1.0`

?

Reply:
It depends what we really want to test.
- If we simply want to check the access to the Resolver domain, then your point is valid.
- If instead we want to make this test closer to the actual use of the Identity Lib against a Resolver, then we should use a valid Twin DID and check we get back `200` as a status code (which we don't get as per your point)

Reply:
I would have a test with HEAD to check access to the domain (which in fact was what ross was needing) 
If you also want to test a valid GET (but I don't see the value of it to be honest) I'd pass the valid twin DID as an additional parameter to the script and execute the test ONLY if there''s a twin to test. We can make sure to give the customer a valid string only when necessary